### PR TITLE
Enable SPCS integration tests

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added or updated automated unit tests to verify correctness of my new code.
   * [ ] I've added or updated integration tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

**Comment by sfc-gh-pjob** [comment]
Can we remove `[connections.spcs]` from TOML configs?

File: tests_integration/testing_utils/snowpark_services_utils.py [hunk 1]
```diff
@@ -73,10 +72,11 @@ def logs_should_return_service_logs(self, service_name: str) -> None:
                 "--schema",
                 self.schema,
             ],
-            connection="spcs",
         )
         assert result.output
-        assert result.output.strip() == f"{service_name}/0 Hello World!"
+        # Assert this instead of full payload due to log coloring
```
Comment (sfc-gh-davwang) [inline tests_integration/testing_utils/snowpark_services_utils.py:77]: What does log coloring mean?

File: tests_integration/scripts/integration_account_setup.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
+SET INT_TEST_USER = 'SNOWCLI_TEST';
+
+-- BASE SETUP
+CREATE ROLE IF NOT EXISTS INTEGRATION_TESTS;
+GRANT CREATE ROLE ON ACCOUNT TO ROLE INTEGRATION_TESTS;
```
Comment (sfc-gh-davwang) [inline tests_integration/scripts/integration_account_setup.sql:5]: should we also include a line for `CREATE USER IF NOT EXISTS IDENTIFIER($INT_TEST_USER);` or do we assume that the person running the tests already has a user they want to use for the tests?

File: tests_integration/containers/docker/bootstrap_containers_setup.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+cd ..
+docker build -t snowpark_test:1 containers/docker
+snow registry token --format=json -c int | docker login
```
Comment (sfc-gh-davwang) [inline tests_integration/containers/docker/bootstrap_containers_setup.sh:3]:
```suggestion
snow registry token --format=json -c integration | docker login YOUR_HOST -u 0sessiontoken --password-stdin
```

File: tests_integration/containers/docker/bootstrap_containers_setup.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+cd ..
+docker build -t snowpark_test:1 containers/docker
```
Comment (sfc-gh-davwang) [inline tests_integration/containers/docker/bootstrap_containers_setup.sh:2]: does the snowpark_test:1 needs the repository url before it?
(e.g. myorg-myacct.registry.snowflake.com/my_db/my_schema/my_repository/snowpark_test:1)

File: tests_integration/scripts/integration_account_setup.sql (hunk 2 above)
Comment (sfc-gh-astus) [reply tests_integration/scripts/integration_account_setup.sql:5]: My bad, I sent only this to Tomek. Here is the missing part:

```
create role integration_tests;
grant role integration_tests to user snowcli_test;

grant create database on account to role integration_tests;
grant create role on account to role integration_tests;
grant usage on warehouse xsmall to role integration_tests;
grant operate on warehouse xsmall to role integration_tests;

grant create application package on account to role integration_tests;
grant create application on account to role integration_tests
```

File: tests_integration/testing_utils/snowpark_services_utils.py (hunk 1 above)
Comment (sfc-gh-turbaszek) [reply tests_integration/testing_utils/snowpark_services_utils.py:77]: The output of snowcli is generate by rich wich has some auto coloring feature. So you have escape codes in the string

File: tests_integration/scripts/integration_account_setup.sql (hunk 2 above)
Comment (sfc-gh-turbaszek) [reply tests_integration/scripts/integration_account_setup.sql:5]: Good point, added!

File: tests_integration/containers/docker/bootstrap_containers_setup.sh (hunk 4 above)
Comment (sfc-gh-turbaszek) [reply tests_integration/containers/docker/bootstrap_containers_setup.sh]: Fixed

**Review by sfc-gh-astus — APPROVED** [review]

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**